packages/site-demo/content/product/components/expansion-panel/react/examples.tsx

```diff
@@ -17,16 +17,22 @@ import {
 
 const App = () => {
     const [isOpen1, setOpen1] = useState(false);
+    const onClose1 = () => setOpen1(false);
     const [isOpen2, setOpen2] = useState(false);
+    const onClose2 = () => setOpen2(false);
     const [isOpen3, setOpen3] = useState(false);
+    const onClose3 = () => setOpen3(false);
     const [isOpen4, setOpen4] = useState(false);
+    const onClose4 = () => setOpen4(false);
     const [isOpen5, setOpen5] = useState(false);
+    const onClose5 = () => setOpen5(false);
     const [isOpen6, setOpen6] = useState(false);
+    const onClose6 = () => setOpen6(false);
     const stopPropagation = (evt: Event) => evt.stopPropagation();
 
     return (
         <>
-            <ExpansionPanel hasBackground hasHeaderDivider isOpen={isOpen1} toggleCallback={setOpen1}>
+            <ExpansionPanel hasBackground hasHeaderDivider isOpen={isOpen1} onClose={onClose1}>
```

Comment:
Maybe we could change all the `onClose` callbacks everywhere to be `onClose(open: false): void` so that we could just do `onClose={setOpen}` ?

Reply:
Good to know!
Won't be necessary since `onToggleOpen` will be finally kept but I'll keep this tip in mind.